### ISSUE TITLE
Add option to change browser.tabs.selectOwnerOnClose

### ISSUE THREAD

**Issue by cpriest** [cpriest/SnapLinksPlus#221]
Since openerTabId is now being set for TreeStyleTabs, in FF it activates a sometimes undesirable feature which is to switch back to the 'opening tab' when an opened tab is closed.

This can be controlled via browser.tabs.selectOwnerOnClose in about:config.  See if this option can be controlled from the options panel from a browser extension.

**Comment by cpriest** [comment]
Relates to #220

**Comment by Keith94** [comment]
AFAIK WebExtensions have no way of reading Firefox preferences. Unless something changed.

**Comment by cpriest** [comment]
Yes, that's what my research has just shown as well.  IDK what really to do about this except to add the option and provide instructions on clicking the option or something.  

Have you seen other extensions have some elegant solution (instructing the user) to this problem?

**Comment by Keith94** [comment]
I don't know, maybe it's possible to make Tree Style Tab support optional via a checkbox?
Otherwise, you could mention that issue in the readme/add-on description.

**Comment by cpriest** [comment]
True, it could be optional.

**Comment by cpriest** [comment]
There is now an interim release being made available which disables the new feature which causes this issue.  The next release will have it behind a disabled-by-default option.

**Comment by thaliel** [comment]
I am not using treestyletabs and browser.tabs.selectOwnerOnClose  ist st to true. Have had tabs open in the desired order for several months now.

**Comment by cpriest** [comment]
Okay
